integreat_cms/release_notes/current/unreleased/2699.yml

```
@@ -0,0 +1,2 @@
+en: fixed the max-width of the feedback object column for admin feedback and region feedback
+de: Vergrößern der max-width der Feedbackspalte für admin feedback und das region feedback
```

```suggestion
de: Vergrößere die maximale Breite der Feedbackspalte für Regionen- und Administrationsfeedback
```
Nit: Not very important, but I think this way it's a bit better to understand for our testers who don't have a technical background. Feel free to reject my suggestion, if you don't agree :blush: